static/js/add_build.js

```diff
@@ -65,6 +65,14 @@ const Features = (() => {
         return 'category_'+category_name.split(" ").join("_");
     }
 
+    function featureIsDiabled(feature_label) {
```

Comment:
```suggestion
    function featureIsDisabled(feature_label) {
```

Reply:
Sorry. I should have double checked it. Thanks for pointing it out.